### PR TITLE
Removing unneeded Node modules and Stripping down Dockerfile for broker

### PR DESCRIPTION
**Pull request by subhankarc**
* Removing unneeded Node Modules from various applications
* Stripping down Dockerfile to have only necessary node modules and applications

### CODE REVIEW

File: broker/Dockerfile [hunk 1]
```diff
@@ -5,16 +5,25 @@ FROM node:12.14-alpine
...
 WORKDIR /opt/service-fabrik-broker/broker
-COPY . ./
-RUN npm install --no-optional && npm cache clean --force
+COPY package.json package.json
+COPY lerna.json lerna.json
+COPY ./applications/osb-broker applications/osb-broker
+COPY ./applications/quota-app applications/quota-app
+COPY ./core core
+COPY ./data-access-layer data-access-layer
+COPY ./config config
```
Comment (subhankarc) [inline broker/Dockerfile:23]: Is config needed ?

Comment (subhankarc) [reply broker/Dockerfile:23]: I guess it is not needed. Can you please remove and test without this ?

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n is succeeded.

**Pull request merged**